datafusion/core/src/datasource/statistics.rs

```diff
@@ -211,49 +199,3 @@ pub(crate) fn get_col_stats(
         })
         .collect()
 }
-
-/// If the given value is numerically greater than the original maximum value,
-/// return the new maximum value with appropriate exactness information.
-fn set_max_if_greater(
```

Comment:
This is the same as https://github.com/apache/arrow-datafusion/blob/eef654c3b0c22b1f845b1441320b8bb718ddd605/datafusion/common/src/stats.rs#L75-L87

Reply:
There is a similar difference here as well. Max values are conserved by relaxing the exactness when an absent statistic is read from the file.